src/tools/rest-backend/Makefile

```diff
@@ -0,0 +1,39 @@
+SOURCES		        = $(wildcard *.cpp)
```

Comment:
Will keep it for testing as it increases development speed a lot. Final version will be cleaned up a bit.